src/lib.rs

```diff
@@ -240,6 +248,52 @@ impl Opts {
   }
 }
 
+#[derive(Default)]
+pub struct WarningOpts {
+  pub all: bool,
```

Comment:
We shouldn't have a `all` option, it should instead activate all the individual options.